src/rbd_replay/ActionTypes.cc

```diff
@@ -26,7 +26,7 @@ void decode_big_endian_string(std::string &str, bufferlist::iterator &it) {
 #if defined(CEPH_LITTLE_ENDIAN)
   uint32_t length;
   ::decode(length, it);
-  length = swab32(length);
```

Comment:
@dillaman the rbd_replay/ActionTypes.cc and rbd-nbd.cc are updated to accommodate the changes in `byteswap.h`, could you help take a look?